### ISSUE TITLE
Usage with Livewire

### ISSUE THREAD

**Issue by abishekrsrikaanth** [spatie/laravel-honeypot#68]
A example of how to use this with Laravel Livewire will be great.

**Comment by freekmurze** [comment]
Currently don't have the time to add an example for this. Feel free to PR one to the docs.

**Comment by vadykoo** [comment]
Please poke give me an example, I can't find

> Currently don't have the time to add an example for this. Feel free to PR one to the docs.

**Comment by jorenvh** [comment]
For anyone else landing on this issue like me. I encountered an issue where a wired field lost focus after every livewire request. After quite some debugging I noticed the issue was gone whenever I removed the honeypot blade directive.

In the end, this made sense because I had the `randomize_name_field_name` set to true in my config. Every time Livewire tried to render the component the honeypot field had a different name.

Two solutions: 

- Disable the `randomize_name_field_name` config setting if you want to keep using the blade directive within the Livewire component
- Move the (in my case) from opening tag including the @csrf & @honypot directive out of the component

Currently my form isn't completed yet so I'm not even sure yet if this will cover everything (when submitting the form) but it did solve my field focus issue. I will update this comment when form handling is finalised, and if need I will create a PR the improve the documentation and/or Livewire support.
